main.js

```diff
@@ -0,0 +1,64 @@
+import Book from './store.js';
+
+const createBook = document.querySelector('#addbook');
+
+class Library {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+  }
+
+  collectBooks(title, author) {
+    const book = new Book(title, author);
+    this.books.push(book);
+    localStorage.setItem('books', JSON.stringify(this.books));
+    this.showBooks();
+  }
+
+  removeBook(title, author) {
+    this.books = this.books.filter((book) => book.title !== title || book.author !== author);
+    localStorage.setItem('books', JSON.stringify(this.books));
+    this.showBooks();
+  }
```

Comment:
- [ ] **Good job so far on making class methods to `add` and `remove` books 👏 👏**. However, the `add` is working fine 👍 But removing a book with duplicated `title` and `author` is removing all the books at once 😨.

   I think it would be better to include the `book id` while pointing the book to be removed 👍 👍 

https://www.loom.com/share/b744a0d949d54ca4a7340ae792771f67